src/api/style.js

```diff
@@ -222,7 +230,7 @@ export default class Style {
         this.filterShader = r.shader;
     }
 
-    _replaceChild(toReplace, replacer) {
+    replaceChild(toReplace, replacer) {
```

Comment:
I don't think we want to expose this implementation detail (this function)

Reply:
IMO this is a great example to use `$` prefix since is used outside `style.js` so is not private (`_` ) but is not part of the public API

Reply:
Hmu... I'm ok with `$` if we all agree on that to make things consistent. Is `$` ok with you @Jesus89 ?

Reply:
We are using this convention in carto.js so :+1:

Reply:
I like the convention too but I think we should address this in another issue since there are lots of wrong underscored/public/private functions

Reply:
:+1: